src/wallet/wallet2.cpp

```diff
@@ -6200,6 +6220,35 @@ void wallet2::get_outs(std::vector<std::vector<tools::wallet2::get_outs_entry>>
     COMMAND_RPC_GET_OUTPUTS_BIN::request req = AUTO_VAL_INIT(req);
     COMMAND_RPC_GET_OUTPUTS_BIN::response daemon_resp = AUTO_VAL_INIT(daemon_resp);
 
+    struct gamma_engine
+    {
+      uint64_t min() const { return 0; }
+      uint64_t max() const { return std::numeric_limits<uint64_t>::max(); }
```

Comment:
`min` and `max` need to be static according to the specification. OSX builds are failing: https://build.getmonero.org/builders/monero-static-osx-10.13/builds/762/steps/compile/logs/stdio